spec/forms/steps/case/hearing_details_form_spec.rb

```diff
@@ -50,7 +50,9 @@
     context 'hearing_date' do
       it_behaves_like 'a multiparam date validation',
                       attribute_name: :hearing_date,
-                      allow_future: true
+                      allow_future: true,
+                      earliest_year: 2010,
+                      latest_year: 2035
```

Comment:
Not sure if the latest year is being tested as expected 🤔 I believe the year is currently [hardcoded to 2051](https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/spec/support/shared_examples/form_validation_shared_examples.rb#L114)

Reply:
You're right! I'll update the shared example to take the `latest_year` into account.